src/c++/perf_analyzer/request_rate_manager.cc

```diff
@@ -130,18 +130,21 @@ RequestRateManager::CreateWorkerSchedules(
 
   std::vector<RateSchedulePtr_t> worker_schedules =
       CreateEmptyWorkerSchedules();
+  std::vector<size_t> thread_ids{CalculateThreadIds()};
```

Comment:
I don't see us actually using the data in this vector(??). We only ever use its size.